src/Common/DefinitionListFormatter.cs

```diff
@@ -51,17 +53,27 @@ public override string ToString()
                     continue;
                 }
 
-                string[] lines = row.Item2.Split(Environment.NewLine);
+                string[] lines = row.Item2.Split(new[] { "\r\n", "\r", "\n" }, StringSplitOptions.None);
```

Comment:
q: is "\r" needed? I'm wondering if it will split "\r\n" twice

Reply:
It won't - it should work fine anyway. Buy I guess it really isn't needed as Mac OS (i.e. before Mac OS X) isn't used anymore